libs/service-portal/information/src/lib/queries/getCompanyRegistryCompany.ts

```diff
@@ -0,0 +1,33 @@
+import { gql } from '@apollo/client'
+
+export const COMPANY_REGISTRY_INFORMATION = gql`
+  query companyRegistryCompanyQuery($input: RskCompanyInfoInput!) {
```

Comment:
I know this is done almost everywhere, but we really shouldn't store queries in a central location and query everything. This takes the power away from GraphQL, and treats it like a Rest API.
Components should query the needed data. GraphQL will take care of merging those requests into a batch operation and remove redundant fields.
For example, this query is making two API requests to external data providers... every single time, regardless of if it's needed or not. This makes field resolvers obsolete, since their purpose are bypassed.
If the component/screen only needs the name and nationalId, then we skip a whole API request to another external API.
In some cases we can be saving 5 external requests using field resolvers... and if that query is a relay like connection that fetches 100 items, then it can save 500 external requests (you get the point 😄 ).
We should take this up in the next dev meeting to solve this globally for service-portal, but IMO this PR should extract the GraphQL AST to the component.

Rule of thumb, if we want to share some GraphQL ASTs, then it should be shared via fragments.

Reply:
Yes, totally agree with you there.
I'll add a ticket on our board to refactor this a bit in the service-portal, and try to make sure this is done properly in the future 😃